src/backend/gl/src/command.rs

```diff
@@ -984,6 +986,14 @@ impl command::RawCommandBuffer<Backend> for RawCommandBuffer {
                             self.push_cmd(Command::BindTexture(*binding, *texture))
                         }
                     }
+                    n::DescSetBindings::Surface(binding, surface) => {
```

Comment:
this shouldn't be needed. A render buffer in GL can only be used for rendering into. You can't create a descriptor set with it, because descriptors are resources, and any operations to them require there to be a full-featured texture instead of a render buffer.